test/intl402/Locale/constructor-options-language-valid.js

```diff
@@ -44,10 +49,15 @@ for (const [language, expected] of validLanguageOptions) {
     `new Intl.Locale('en-US', {language: "${language}"}).toString() returns "${expect}"`
   );
 
-  expect = expected || 'en-els';
-  assert.sameValue(
-    new Intl.Locale('en-els', {language}).toString(),
-    expect,
-    `new Intl.Locale('en-els', {language: "${language}"}).toString() returns "${expect}"`
-  );
+  assert.throws(RangeError, () => new Intl.Locale('en-els', {language}).toString());
+
+}
+
+const invalidLanguageOptions = [
+    'abcd',
+];
+for (const language of invalidLanguageOptions) {
+  assert.throws(RangeError, () => new Intl.Locale('en', {language}).toString());
+  assert.throws(RangeError, () => new Intl.Locale('en-US', {language}).toString());
+  assert.throws(RangeError, () => new Intl.Locale('en-els', {language}).toString());
```

Comment:
no need for the toString() in these calls, this code is unreachable.

Reply:
done